_posts/2017-07-25-editor.md

```diff
@@ -19,18 +19,16 @@ I started this blog in May of last year. I don't mean to brag, but by last April
 
 The size of my audience changed a bit in May 2017. The sharp-eyed reader may be able to spot the subtle shift in my traffic graph:
 
-{% capture fig_img %}
-[![Blog traffic graphs](/images/2017-07-25-editor/full-traffic-graph-sm.png){: .align-center}](/images/2017-07-25-editor/full-traffic-graph.png)
-{% endcapture %}
-
-{% capture fig_caption %}
-Number of unique readers visiting [mtlynch.io](https://mtlynch.io) per week
-{% endcapture %}
-
-<figure>
-  {{ fig_img | markdownify | remove: "<p>" | remove: "</p>" }}
-  <figcaption>{{ fig_caption | markdownify | remove: "<p>" | remove: "</p>" }}</figcaption>
-</figure>
+{% assign fig_caption = "Number of unique readers visiting [mtlynch.io](https://mtlynch.io) per week" | markdownify | remove: "<p>" | remove: "</p>" %}
+
+{% responsive_image_block %}
+    path: images/2017-07-25-editor/full-traffic-graph.png
```

Comment:
I think it's a pretty safe bet that all images will be within `images/` so can we assume paths are relative to that directory? 

Actually, is it possible to infer automatically that they'll be in `images/{{ filename w/o extension }}/foo.png`? It would be great if I could just write `path: foo.png` and it would know to find it in `images/2017-07-25-editor/foo.png`.